automation_tools/__init__.py

```diff
@@ -1286,34 +1286,54 @@ def configure_osp(admin_password=None, forward_zone=None, reverse_zone=None,
     ADMIN_PASSWORD
         The Satellite6 Admin password.
 
+    SATELLITE_VERSION
+        Version of the satellite that is installed.(Eg. 6.2,6.3)
+
     """
+    satellite_version = os.environ.get('SATELLITE_VERSION')
     if reverse_zone is None:
         reverse_zone = os.environ.get('OSP_REVERSE_ZONE')
     if forward_zone is None:
         forward_zone = os.environ.get('OSP_FORWARD_ZONE')
-    if template_url is None:
-        template_url = os.environ.get('OSP_FINISH_TEMPLATE_URL')
     if admin_password is None:
         admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
-    run('cp /etc/zones.conf /root/zones-pre_osp.conf')
-    run('for i in {0}; do satellite-installer '
-        '--foreman-proxy-dns-reverse $i.in-addr.arpa ; '
-        'sed -n 1,7p /etc/zones.conf >> /root/zones_cons.conf ; done'
-        .format(reverse_zone))
-    run('satellite-installer --foreman-proxy-dns-zone {0}'
-        .format(forward_zone))
-    run('sed -n 8,14p /etc/zones.conf >> /root/zones_cons.conf')
-    run('cat /root/zones_cons.conf > /etc/zones.conf')
-    run('cat /root/zones-pre_osp.conf >> /etc/zones.conf')
-    run('hammer -u admin -p changeme template clone '
-        '--name \'Satellite Kickstart Default Finish\' '
-        '--new-name \'Satellite Kickstart Default Finish OSP\'')
-    run('wget -O /root/Satellite_Kickstart_Default_Finish_OSP.txt {0}'
-        .format(template_url))
-    run('hammer -u admin -p {0} template update --name '
-        '\'Satellite Kickstart Default Finish OSP\' --type finish '
-        '--file /root/Satellite_Kickstart_Default_Finish_OSP.txt'
-        .format(admin_password))
+    if template_url is None:
+        template_url = os.environ.get('OSP_FINISH_TEMPLATE_URL')
+    run('rm -rf /root/zones_cons.conf', warn_only=True)
+    run('rm -rf /root/zones-pre_osp.conf', warn_only=True)
+    if satellite_version == '6.2':
+        run('cp /etc/zones.conf /root/zones-pre_osp.conf')
```

Comment:
On second thought , I feel it should actually be
```
if satellite_version == '6.1' and  satellite_version == '6.2':
    zone_file = /etc/zones.conf
else:
    zone_file = /etc/named/zones.conf
```
As, it would also be of use for downstream-nightly and upstream-nightly setups.